<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkudDialog</class>
 <widget class="QDialog" name="SkudDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>240</width>
     <height>240</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>371</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Set settings</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxUp">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxRight">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Rigtht</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxDown">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxLeft">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
